pom.xml

```diff
@@ -231,6 +231,21 @@
             <artifactId>swagger-annotations</artifactId>
             <version>2.2.2</version>
         </dependency>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.gsonfire</groupId>
+            <artifactId>gson-fire</artifactId>
+            <version>1.8.5</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
```

Comment:
This adds a new HTTP client to the library, can we remove this dependency via templates? Otherwise, it's better to migrate `--library` to `apache-httpclient` to match our current client.

Reply:
I can probably narrow it down to the classes the library models use from this package.